pagopa/pn-frontend
# PN-4432-PG-drop-down-app-menu

## Short description
For pg-webapp, now the name of the app at the top-left corner has been replaced by a drop-down menu with the options of SelfCare and PN app dashboards, as indicated by the corresponding Figma.

## List of changes proposed in this pull request
- Use the productsList option of the mui-italia Layout to make it show the drop-down menu.
- Use the value of REACT_APP_URL_SELFCARE_BASE to define the SelfCare dashboard URL.
- Feature B

## How to test
Launch PG webapp, verify the existence and behavior of the drop-down menu at top-left.
